lib/connect-memcached.js

```diff
@@ -110,16 +118,15 @@ module.exports = function(session) {
       var maxAge = sess.cookie.maxAge;
       var ttl =
         this.ttl || ("number" == typeof maxAge ? (maxAge / 1000) | 0 : oneDay);
-      var sess = JSON.stringify(
-        this.secret
-          ? encryptData.call(
-              this,
-              JSON.stringify(sess),
-              this.secret,
-              this.algorithm
-            )
-          : sess
-      );
+
+      if (this.client.secret) {
+        this.kruptein.set(this.client.secret, JSON.stringify(sess), function(err, ct) {
```

Comment:
In the `get` call, `this.secret` was reassigned. If I remember correctly it was to avoid local scoping issues where the `set` method didn’t have them so I reference it directly.

One thing I should mention is that this PR was provided for `v2.x`, the library is now at `v3.x`.

would you prefer I resubmit?

Reply:
If you would be so kind, please do :) Thank you

Reply:
Also guessing you'll find useful tests added recently to the repo.